test/unit/bootloader/install/grub2_test.py

```diff
@@ -448,8 +449,19 @@ def side_effect(device, mountpoint=None):
             filename='shim-install', root_dir='tmp_root'
         )
 
+    @patch('kiwi.bootloader.install.grub2.Command.run')
+    @patch('kiwi.bootloader.install.grub2.MountManager')
     @patch('os.path.isfile')
-    def test_set_disk_password(self, mock_os_path_is_file):
+    def test_set_disk_password(
+        self, mock_os_path_is_file, mock_mount_manager, mock_command
```

Comment:
Since you optimized out `os.path.isfile()`, `mock_os_path_is_file` is no longer needed, or?

Reply:
right, this mock can be deleted from the test

Reply:
done